File: server/services/student-service/student/src/main/java/com/saferide/student/application/event/StudentCreated.java
package com.saferide.student.application.event;

import java.time.Instant;
import java.util.UUID;

public record StudentCreated(
        UUID studentId,
        UUID schoolId,
        String firstName,
        String lastName,
        String parentFirstName,
        String parentLastName,
        String parentEmail,
        String parentPhone,
        Instant occurredAt) {}

File: server/services/identity-service/SafeRide.Identity/SafeRide.Identity.Application/Events/StudentCreated.cs
namespace SafeRide.Identity.Application.Events;

public sealed record StudentCreated(
    Guid StudentId,
    Guid SchoolId,
    string FirstName,
    string LastName,
    string ParentFirstName,
    string ParentLastName,
    string ParentEmail,
    string ParentPhone,
    DateTime OccurredAt
);
